csharp/src/Drivers/Apache/Hive2/HiveServer2Statement.cs

```diff
@@ -139,16 +144,21 @@ public class Options
             // Options common to all HiveServer2Statement-derived drivers go here
             public const string PollTimeMilliseconds = "adbc.statement.polltime_milliseconds";
```

Comment:
All of the options should be consistent across the namespaces. For this driver, it might be a little different than others, but I would think something like:

`adbc.apache.statement.polltime_milliseconds` if it is generic to all of the drivers

`adbc.apache.spark.convert_float` if it is specific to one of the drivers

Reply:
My vote would be to match the C# namespace structure, where `adbc.apache.statement.*` is generic to all 3 drivers, but `adbc.apache.spark.statement` or `adbc.apache.hiveserver2.statement` would be specific to Spark and Hive.